src/sparse_variational.jl

```diff
@@ -1,16 +1,72 @@
+raw"""
+    Centred()
+
+Used in conjunction with `SparseVariationalApproximation`.
+States that the `q` field of [`SparseVariationalApproximation`](@ref) is to be interpreted
+directly as the approximate posterior over the pseudo-points.
+
+This is also known as the "unwhitened" parametrisation [1].
+
+See also [`NonCentred`](@ref).
+
+[1] - https://en.wikipedia.org/wiki/Whitening_transformation
 """
-    SparseVariationalApproximation(fz::FiniteGP, q::AbstractMvNormal)
+struct Centred end
```

Comment:
Has there been a decision before on whether to stick to British or American English around JuliaGPs?

Reply:
I was always puzzled by the Centred vs Centered but in the paper on non centred parameterization we refer to, they also write it this way

Reply:
that'd be one advantage of Whitened vs NonWhitened :joy:

Reply:
I would have a preference for American English (seems to be more common all around; e.g. how it's Colors.jl not Colours.jl), and a strong preference for within-package consistency.

Reply:
Unsurprisingly, I have a reasonably strong preference for using British English (I caught myself saying "math" the other day 😬 ). If there is a majority of contributors in favour of using American English I'll go along with it though.

Having said I try to use it "where possible", I'm not sure there are any instances of American English in this package at the minute, so I'm not sure that consistency is a problem at the minute.

Reply:
I also prefer British (definitely not biased :smile: ). I agree it's a good reason to use Whitened instead of Centred though - would be better to keep the API dialect agnostic

Reply:
> I'm not sure that consistency is a problem at the minute.

hopefully not, I just wanted to make sure that this is a discussion we're having before it becomes an issue: )

Reply:
> I agree it's a good reason to use Whitened instead of Centred though - would be better to keep the API dialect agnostic

I think my preference is still centred vs non-centred rather than un-whitened vs whitened, because the former is very standard terminology throughout the hierarchical modelling world in general (I believe the [earliest use of the term was in 1995](https://www.jstor.org/stable/pdf/27645805.pdf?casa_token=XQnTIWTxgEUAAAAA:gYTIwc-z3mKjS1t1ApF4rp2YmfkbgiWaVaIcfpXAVmP6DfKygyC2ZE0_8uJQ1z5ml5S2csD6tBsO_May9JvwQB_9PJDOtWKA11aBxo4EcNniuzzK-A)), of which we are a part.

Reply:
> In any case GPs are historically quite British. This could be justified this way laughing To avoid any confusion we can have `Centered` and `NonCentered` throw an error like "You muppet! Use proper English _insert chosen British swear word_"

Sounds great :+1:  Let's do that! :joy: 

On a more serious note, while I'd prefer `Centered` over `Centred`, we can go with the latter, but I would then suggest clarifying (e.g. in the README) that this repo uses British English?

Now I'm wondering if that means if we define some optimisation utils we should call them `optimise` even though it's `Optim.optimize`.........

Reply:
Eurgh. Having slept on it a bit, I'm concerned that using British English is going to be frustrating for anyone that's not from the UK / hasn't spent a lot of time here listening to us moan about American English, which is most of the world.

Let's discuss this tomorrow at our regular meeting. I saddens me greatly to say that I think we might be best of adopting American English throughout JuliaGPs.

Reply:
I mean, it would force me to learn more French, which is something I've been meaning to do since school...